Apps/Common/src/Common.csproj

```diff
@@ -9,6 +9,7 @@
         <PackageReference Include="Hangfire.AspNetCore" Version="1.7.33" />
         <PackageReference Include="Hangfire.PostgreSql" Version="1.9.9" />
         <PackageReference Include="Hellang.Middleware.ProblemDetails" Version="6.5.1" />
+        <PackageReference Include="libphonenumber-csharp" Version="8.13.6" />
```

Comment:
This package probably should not be here and instead in the UI projects

Reply:
Its currently used by the phone number validator. I've moved this to GatewayAPI along with the validator and its tests.